src/cmds/core/other.py

```diff
@@ -32,8 +32,8 @@ async def callback(self, interaction: discord.Interaction):
         
         message_body = self.children[1].value
         # Slack has no way to disallow @(@everyone calls), so we strip it out and replace it with a safe version
-        title = title.replace("@", "[at]") 
-        message_body = message_body.replace("@", "[at]") 
+        title = title.replace("@", "[at]").replace("<", "[bracket]") 
+        message_body = message_body.replace("@", "[at]").replace("<", "[bracket]") 
```

Comment:
@0xEmma do we need to replace also `>`?

Reply:
Just doing < would cover it, we _could_ do both, but just doing one would still mitigate